src/main/kotlin/mate/academy/ColorSupplier.kt

```diff
@@ -0,0 +1,14 @@
+package mate.academy
+
+import kotlin.random.Random
+
+open class ColorSupplier {
+
+
+    companion object {
+        fun getRandomColor(): String {
+            val index = Random.nextInt(Colors.values().size)
+            return Colors.values()[index].toString()
+        }
```

Comment:
This method "getRandomColor" could be part of Colors enum class, as extension function, would be more comfortable to use